src/App.tsx

```diff
@@ -1,14 +1,55 @@
-/* eslint-disable max-len */
-import React from 'react';
+import { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { FilterTypes } from './types/FilterTypes';
 
 export const App: React.FC = () => {
+  const [todo, setTodo] = useState<Todo | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(FilterTypes.all);
+  const [query, setQuery] = useState('');
+  const [loadedTodos, setLoadedTodos] = useState<Todo[]>(todos);
+
+  const compareLower = (title: string) => (
+    title.toLowerCase().includes(query.toLowerCase())
+  );
+
+  const isLoading = !todos.length;
```

Comment:
What if the server will return an empty array?

Reply:
need create state isLoading with default value true and set it to false when we get a response from the server